Make compatible with fastapi>=0.89

Changes in Response from 0.89 causing following error which needs to be fixed
```
fastapi.exceptions.FastAPIError: Invalid args for response field! Hint: check that <class 'starlette.responses.Response'> is a valid pydantic field type
Error: Process completed with exit code 1.
```

--- a/.github/workflows/typing.yml
+++ b/.github/workflows/typing.yml
@@ -27,7 +27,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         python -m pip install .[test]
-        python -m pip install starlette==0.20.4
+        python -m pip install starlette==0.25.0
     - name: Run mypy
       run: |
         mypy -p ert_storage --ignore-missing-imports --disallow-untyped-defs --show-error-codes

--- a/.github/workflows/typing.yml
+++ b/.github/workflows/typing.yml
@@ -30,4 +30,4 @@ jobs:
         python -m pip install starlette==0.25.0
     - name: Run mypy
       run: |
-        mypy -p ert_storage --ignore-missing-imports --disallow-untyped-defs --show-error-codes
+        mypy -p ert_storage --ignore-missing-imports --disallow-untyped-defs --show-error-codes --exclude _alembic --exclude testing

--- a/setup.py
+++ b/setup.py
@@ -56,7 +56,7 @@ setup(
     },
     install_requires=[
         "alembic",
-        "fastapi<0.89",
+        "fastapi",
         "httpx",
         "numpy",
         "pandas",

--- a/src/ert_storage/endpoints/ensembles.py
+++ b/src/ert_storage/endpoints/ensembles.py
@@ -14,7 +14,6 @@ router = APIRouter(tags=["ensemble"])
 def post_ensemble(
     *, db: Session = Depends(get_db), ens_in: js.EnsembleIn, experiment_id: UUID
 ) -> ds.Ensemble:
-
     experiment = db.query(ds.Experiment).filter_by(id=experiment_id).one()
     active_reals = (
         ens_in.active_realizations

--- a/src/ert_storage/endpoints/updates.py
+++ b/src/ert_storage/endpoints/updates.py
@@ -12,7 +12,6 @@ def create_update(
     db: Session = Depends(get_db),
     update: js.UpdateIn,
 ) -> js.UpdateOut:
-
     ensemble = db.query(ds.Ensemble).filter_by(id=update.ensemble_reference_id).one()
     update_obj = ds.Update(
         algorithm=update.algorithm,

--- a/tests/integration/test_azure_storage.py
+++ b/tests/integration/test_azure_storage.py
@@ -47,7 +47,6 @@ def test_blob(client, azure_client, simple_ensemble):
 
 
 def test_blocked_blob(client, azure_client, simple_ensemble):
-
     ensemble_id = simple_ensemble()
 
     size = 12 * 1024**2

--- a/tests/integration/test_records.py
+++ b/tests/integration/test_records.py
@@ -452,7 +452,6 @@ def test_forward_model_file(client, simple_ensemble):
 
 @pytest.mark.skip(reason="github.com/equinor/ert-storage/issues/234")
 def test_chunked_blob(client, simple_ensemble):
-
     ensemble_id = simple_ensemble()
 
     size = 12 * 1024**2

